packages/scaffold-ui/src/modal/w3m-modal/index.ts

```diff
@@ -39,19 +38,24 @@ export class W3mModal extends LitElement {
 
   @state() private loading = ModalController.state.loading
 
+  @state() private shake = ModalController.state.shake
+
   public constructor() {
     super()
     this.initializeTheming()
     ApiController.prefetch()
     this.unsubscribe.push(
-      ModalController.subscribeKey('open', val => (val ? this.onOpen() : this.onClose())),
-      ModalController.subscribeKey('loading', val => {
-        this.loading = val
-        this.onNewAddress(AccountController.state.caipAddress)
-      }),
-      AccountController.subscribeKey('isConnected', val => (this.connected = val)),
-      AccountController.subscribeKey('caipAddress', val => this.onNewAddress(val)),
-      OptionsController.subscribeKey('isSiweEnabled', val => (this.isSiweEnabled = val))
+      ...[
+        ModalController.subscribeKey('open', val => (val ? this.onOpen() : this.onClose())),
+        ModalController.subscribeKey('shake', val => (this.shake = val)),
+        ModalController.subscribeKey('loading', val => {
+          this.loading = val
+          this.onNewAddress(AccountController.state.caipAddress)
+        }),
```

Comment:
you are subscribing to all props separately, maybe do a `subscribe` instead? 
Or would that generate weird UI behaviors?

Reply:
Since there is no another state, this might make sense yeah. Generally `subscribeKey` is better approach but in this case we could subs to the state directly 👍🏽

Reply:
Actually, this makes the code more readable. When you move all the conditions inside the block with if/else's etc. It'll increase the LOC.